include/livox_laser_simulation/livox_points_plugin.h

```diff
@@ -77,7 +77,7 @@ class LivoxPointsPlugin : public RayPlugin {
  private:
     enum PointCloudType {
         SENSOR_MSG_POINT_CLOUD = 0,
-        SENSOR_MSG_POINT_CLOUD2_POINTXYZ = 1,
+        SENSOR_MSG_POINT_CLOUD2_POINTXYZI = 1,
```

Comment:
Instead of replacing this XYZ point enum, could we instead add another value for the XYZI points?

Reply:
Yes, I'm trying to match the driver output. Check the [README](https://github.com/oxin-ros/livox_ros_driver/tree/master), it says:

> The standard pointcloud2 (pcl :: PointXYZI) format in the PCL library

I also verified with one bag file that I have with me.